src/data/sparse_page_source.h

```diff
@@ -230,30 +264,19 @@ class SparsePageSource : public DataSource<T> {
         writer.PushWrite(std::move(page));
       }
 
-      std::unique_ptr<dmlc::Stream> fo(
-          dmlc::Stream::Create(name_info.c_str(), "w"));
+      std::unique_ptr<dmlc::Stream> fo(dmlc::Stream::Create(cinfo.name_info.c_str(), "w"));
       int tmagic = kMagic;
       fo->Write(&tmagic, sizeof(tmagic));
       // Either every row has query ID or none at all
       CHECK(qids.empty() || qids.size() == info.num_row_);
       info.SaveBinary(fo.get());
     }
-    LOG(INFO) << "SparsePageSource::CreateRowPage Finished writing to "
-              << name_info;
+    LOG(INFO) << "SparsePageSource::CreateRowPage Finished writing to " << cinfo.name_info;
   }
 
   /*!
    * \brief Create source cache by copy content from DMatrix.
-   * \param cache_info The cache_info of cache file location.
-   */
-  static void CreateRowPage(DMatrix* src,
```

Comment:
why is the row page creation removed? is it renamed as something - cached row page?

Reply:
This version is not being used.

Reply:
@rongou Thanks for cleaning up the code.